src/services/homework/hooks/index.js

```diff
@@ -380,15 +352,6 @@ exports.before = () => {
 				hasPatchPermission,
 			]),
 		],
-		remove: [
-			iff(isProvider('external'), [
```

Comment:
remove still exists for feathers, is just that we have no action for it, and it passes simply to mongoose which can do db opperations directly. We should try to disable it
remove: [disallow()],